pyaerocom/colocation_auto.py

```diff
@@ -920,22 +893,10 @@ def _run_gridded_gridded(self, var_name=None):
         obs_reader = ReadGridded(self.obs_id)
 
         if 'obs_filters' in self:
-            remaining_filters = self._eval_obs_filters()
-            if bool(remaining_filters):
-                raise NotImplementedError('Cannot apply filters {} to gridded '
-                                          'observation data.'.format(remaining_filters))
+            obs_filters = self._eval_obs_filters()
```

Comment:
Could do: `obs_filters = self._eval_obs_filters() if 'obs_filters' in self else {}` to ensure `obs_filters` is set.

Reply:
Done now, had some conflicts in colocation_auto.py after merging v090dev, and had to fix manually. Hope everything still works.